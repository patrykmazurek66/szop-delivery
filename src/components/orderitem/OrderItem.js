import Div from "../div/Div";
import P from "../p/P";
import "./orderitem.css";

export default function OrderItem({ qty, name, price }) {
  return (
    <Div full nomp dark>
      <P>
        {qty} Ã— {name}
      </P>
      <P>{(qty * price).toFixed(2)}$</P>
    </Div>
  );
}
