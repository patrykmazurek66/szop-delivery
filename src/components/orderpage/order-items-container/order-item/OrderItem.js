import "./orderitem.css";

export default function OrderItem({ qty, name, price }) {
  return (
    <div className="order-item">
      <p>
        {qty} Ã— {name}
      </p>
      <p>{(qty * price).toFixed(2)}$</p>
    </div>
  );
}
