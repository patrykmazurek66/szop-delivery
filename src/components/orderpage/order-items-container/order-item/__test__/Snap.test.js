import renderer from "react-test-renderer";
import OrderItem from "../OrderItem";

it("renders correctly", () => {
  const tree = renderer.create(<OrderItem />).toJSON();
  expect(tree).toMatchSnapshot();
});
