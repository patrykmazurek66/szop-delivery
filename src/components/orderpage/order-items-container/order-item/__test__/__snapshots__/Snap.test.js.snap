// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="order-item"
>
  <p>
     Ã— 
  </p>
  <p>
    NaN
    $
  </p>
</div>
`;
